registration/src/main/java/org/spine3/samples/lobby/registration/conference/ConferenceProjection.java

```diff
@@ -196,19 +194,17 @@ private void sendRemoveSeatsRequest(SeatTypeId seatTypeId, int quantity) {
                                                .build();
         final Command command = create(message, newCommandContext());
         commandBus.post(command, new StreamObserver<Response>() {
```

Comment:
It is much better to add a class-level comment and explain the `StreamObserver` aim.